### ISSUE TITLE
JDK 8 crashes on Windows in debug mode since J9 vm version 0.46.0 

### ISSUE THREAD

**Issue by MNeuling** [eclipse-openj9/openj9#20133]
Java -version output
--------------------
openjdk version "1.8.0_422"
IBM Semeru Runtime Open Edition (build 1.8.0_422-b05)
Eclipse OpenJ9 VM (build openj9-0.46.0, JRE 1.8.0 Windows 10 amd64-64-Bit Compressed References 20240802_1019 (JIT enabled, AOT enabled)
OpenJ9   - 1a6f6128aa
OMR      - 840a9adba
JCL      - a75ff73ce5 based on jdk8u422-b05)

Summary of problem
------------------
Since JDK 8 update 422 J9 vm 0.46.0 a unit test crashes hard in debug mode with core dump.

Diagnostic files
----------------
```
16:35:12,182 WARN  [VMWideSingleAction] Running CargoSchemaGeneratorAction - all other VMs must wait - socket 10.000, pid 21784@P00328
16:35:12,242 WARN  [VMWideSingleAction] CargoSchemaGeneratorAction finished - all other VMs may run now - pid 21784@P00328
Unhandled exception
Type=Segmentation error vmState=0x000501ff
Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00007FF9307356BD ContextFlags=0010005f
Handler1=00007FF93013EE60 Handler2=00007FF972B0AC60 InaccessibleReadAddress=0000000000000008
RDI=000000000041A800 RSI=00000000030BE0C0 RAX=0000000000000000 RBX=000000000041A7EC
RCX=00000000030BE0C0 RDX=000001E0AD0E40B0 R8=0000000000000008 R9=0000000000025B00
R10=00000000FFFFFFE8 R11=0000001B182F66A0 R12=0000000000000000 R13=000001E0B65DB9E0
R14=00000000030BE0C0 R15=0000000000000000
RIP=00007FF9307356BD RSP=0000001B182F6670 RBP=0000001B182F66F0 EFLAGS=0000000000010206
FS=0053 ES=002B DS=002B
XMM0=0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM1=7c1662d8fb0452b9 (f: 4211364608.000000, d: 5.453995e+289)
XMM2=00007ff4614a13f0 (f: 1632244736.000000, d: 6.950890e-310)
XMM3=000000003f400000 (f: 1061158912.000000, d: 5.242822e-315)
XMM4=0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM5=0000001b182f8db0 (f: 405769664.000000, d: 5.749436e-313)
XMM6=0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM7=00000000461c4000 (f: 1176256512.000000, d: 5.811479e-315)
XMM8=000000003ecccccd (f: 1053609152.000000, d: 5.205521e-315)
XMM9=000000003dcccccd (f: 1036831936.000000, d: 5.122630e-315)
XMM10=0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM11=0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM12=0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM13=0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM14=0000000000000000 (f: 0.000000, d: 0.000000e+00)
XMM15=0000000000000000 (f: 0.000000, d: 0.000000e+00)
Module=C:\Program Files\Java\java8_j9\jre\bin\default\j9jit29.dll
Module_base_address=00007FF9300D0000 Offset_in_DLL=00000000006656bd

Method_being_compiled=com/dakosy/reflections/MethodUtil.toDeclaredMethod(Ljava/lang/Class;Ljava/lang/reflect/Method;)Ljava/lang/reflect/Method;
Target=2_90_20240802_1019 (Windows 10 10.0 build 19045)
CPU=amd64 (16 logical CPUs) (0x7ea44a000 RAM)
----------- Stack Backtrace -----------
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0x59a48d (0x00007FF9307356BD [j9jit29+0x6656bd])
j9jit_testarossa+0x1113c (0x00007FF930168CCC [j9jit29+0x98ccc])
j9jit_testarossa+0x12135 (0x00007FF930169CC5 [j9jit29+0x99cc5])
j9jit_testarossa+0x17e4e (0x00007FF93016F9DE [j9jit29+0x9f9de])
j9jit_testarossa+0x2a3b2 (0x00007FF930181F42 [j9jit29+0xb1f42])
j9jit_testarossa+0x7814 (0x00007FF93015F3A4 [j9jit29+0x8f3a4])
j9jit_testarossa+0x76c6 (0x00007FF93015F256 [j9jit29+0x8f256])
j9jit_testarossa+0x297a1 (0x00007FF930181331 [j9jit29+0xb1331])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xebb21 (0x00007FF930286D51 [j9jit29+0x1b6d51])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xc93f9 (0x00007FF930264629 [j9jit29+0x194629])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xeba4a (0x00007FF930286C7A [j9jit29+0x1b6c7a])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0x6e02d (0x00007FF93020925D [j9jit29+0x13925d])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xe82a4 (0x00007FF9302834D4 [j9jit29+0x1b34d4])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xe7140 (0x00007FF930282370 [j9jit29+0x1b2370])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xe31fd (0x00007FF93027E42D [j9jit29+0x1ae42d])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xdf690 (0x00007FF93027A8C0 [j9jit29+0x1aa8c0])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xdc690 (0x00007FF9302778C0 [j9jit29+0x1a78c0])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0x6de38 (0x00007FF930209068 [j9jit29+0x139068])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xc6487 (0x00007FF9302616B7 [j9jit29+0x1916b7])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xc42dc (0x00007FF93025F50C [j9jit29+0x18f50c])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0x2f5961 (0x00007FF930490B91 [j9jit29+0x3c0b91])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0xc38e6 (0x00007FF93025EB16 [j9jit29+0x18eb16])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0x3b1c61 (0x00007FF93054CE91 [j9jit29+0x47ce91])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0x3b06d9 (0x00007FF93054B909 [j9jit29+0x47b909])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0x3ae5f5 (0x00007FF930549825 [j9jit29+0x479825])
Java_java_lang_invoke_ThunkTuple_initialInvokeExactThunk+0x24b77e (0x00007FF9303E69AE [j9jit29+0x3169ae])
(0x00007FF93012C4E4 [j9jit29+0x5c4e4])
(0x00007FF93012FAF5 [j9jit29+0x5faf5])
j9port_isCompatible+0x18b16 (0x00007FF972B0B6F6 [j9prt29+0x1b6f6])
j9port_isCompatible+0x1a250 (0x00007FF972B0CE30 [j9prt29+0x1ce30])
(0x00007FF93012BC53 [j9jit29+0x5bc53])
(0x00007FF930132035 [j9jit29+0x62035])
(0x00007FF930131A1B [j9jit29+0x61a1b])
(0x00007FF93013EC90 [j9jit29+0x6ec90])
j9port_isCompatible+0x1a28b (0x00007FF972B0CE6B [j9prt29+0x1ce6b])
(0x00007FF93013E986 [j9jit29+0x6e986])
omrthread_get_category+0xa32 (0x00007FF999F44202 [j9thr29+0x4202])
_configthreadlocale+0x92 (0x00007FF9B5751BB2 [ucrtbase+0x21bb2])
Ordinal76+0x11e7 (0x000000007D1911E7 [cyinjct+0x11e7])
BaseThreadInitThunk+0x14 (0x00007FF9B79F7374 [KERNEL32+0x17374])
RtlUserThreadStart+0x21 (0x00007FF9B7CDCC91 [ntdll+0x4cc91])
---------------------------------------
JVMDUMP039I Speicherauszugsereignis "gpf", Detail "" um 2024/09/09 16:35:12 - bitte warten.
JVMDUMP032I JVM forderte als Antwort auf ein Ereignis einen Speicherauszug von System mit "E:\git\shaire_dev\shaire\server-integration\core.20240909.163512.21784.0001.dmp" an
JVMDUMP010I Speicherauszug von System in E:\git\shaire_dev\shaire\server-integration\core.20240909.163512.21784.0001.dmp geschrieben
JVMDUMP032I JVM forderte als Antwort auf ein Ereignis einen Speicherauszug von Java mit "E:\git\shaire_dev\shaire\server-integration\javacore.20240909.163512.21784.0002.txt" an
JVMDUMP010I Speicherauszug von Java in E:\git\shaire_dev\shaire\server-integration\javacore.20240909.163512.21784.0002.txt geschrieben
JVMDUMP032I JVM forderte als Antwort auf ein Ereignis einen Speicherauszug von Snap mit "E:\git\shaire_dev\shaire\server-integration\Snap.20240909.163512.21784.0003.trc" an
JVMDUMP010I Speicherauszug von Snap in E:\git\shaire_dev\shaire\server-integration\Snap.20240909.163512.21784.0003.trc geschrieben
JVMDUMP032I JVM forderte als Antwort auf ein Ereignis einen Speicherauszug von JIT mit "E:\git\shaire_dev\shaire\server-integration\jitdump.20240909.163512.21784.0004.dmp" an
JVMDUMP051I Speicherauszug von JIT in Thread 'JIT Compilation Thread-000' aufgetreten 0x0000000000025B00
JVMDUMP049I Der JIT-Speicherauszug hat alle wartenden Threads �ber die aktuelle zu kompilierende Methode benachrichtigt
JVMDUMP054I JIT-Speicherauszug erstellt einen Trace f�r IL der Methode im abgest�rzten Kompilierungsthread
JVMDUMP048I Die zurzeit kompilierte JIT-Speicherauszugsmethode ist eine normale Methode
JVMDUMP053I JIT-Speicherauszug kompiliert com/dakosy/reflections/MethodUtil.toDeclaredMethod(Ljava/lang/Class;Ljava/lang/reflect/Method;)Ljava/lang/reflect/Method; neu
JVMDUMP052I Rekursiver Absturz des JIT-Speicherauszugs im Diagnosethread aufgetreten
JVMDUMP010I Speicherauszug von JIT in E:\git\shaire_dev\shaire\server-integration\jitdump.20240909.163512.21784.0004.dmp geschrieben
JVMDUMP013I Speicherauszugsereignis "gpf", Detail "" wurde verarbeitet.
```
[javacore.20240909.163512.21784.0002.txt](https://github.com/user-attachments/files/16932229/javacore.20240909.163512.21784.0002.txt)
[jitdump.20240909.163512.21784.0004.dmp](https://github.com/user-attachments/files/16932237/jitdump.20240909.163512.21784.0004.dmp)

OutOfMemoryError: Java Heap Space
---------------------------------
[verbosegc.20240909.163429.21784.txt](https://github.com/user-attachments/files/16932224/verbosegc.20240909.163429.21784.txt)

**Comment by MNeuling** [comment]
[core.20240909.163512.21784.0001.zip.001.zip](https://github.com/user-attachments/files/16933428/core.20240909.163512.21784.0001.zip.001.zip)
[core.20240909.163512.21784.0001.zip.002.zip](https://github.com/user-attachments/files/16933434/core.20240909.163512.21784.0001.zip.002.zip)
[core.20240909.163512.21784.0001.zip.003.zip](https://github.com/user-attachments/files/16933439/core.20240909.163512.21784.0001.zip.003.zip)
[core.20240909.163512.21784.0001.zip.004.zip](https://github.com/user-attachments/files/16933443/core.20240909.163512.21784.0001.zip.004.zip)
[core.20240909.163512.21784.0001.zip.005.zip](https://github.com/user-attachments/files/16933445/core.20240909.163512.21784.0001.zip.005.zip)
[core.20240909.163512.21784.0001.zip.006.zip](https://github.com/user-attachments/files/16933447/core.20240909.163512.21784.0001.zip.006.zip)
[core.20240909.163512.21784.0001.zip.007.zip](https://github.com/user-attachments/files/16933449/core.20240909.163512.21784.0001.zip.007.zip)

**Comment by dmitripivkine** [comment]
@hzongaro FYI Crash during compilation

**Comment by MNeuling** [comment]
Everything works fine in debug mode when applying vm option **`-XX:+EnableExtendedHCR`**.
 Therefore, changes of #19554 seem to cause the issue.

**Comment by hzongaro** [comment]
This is expected to be fixed in the [OpenJ9 0.46.1 release](https://github.com/eclipse-openj9/openj9/releases/tag/openj9-0.46.1)

**Comment by krismarc** [comment]
Could anyone confirm if this is the same? I'm going to try the recent release.

```
ERR #0: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x92fea5) [0x7f2b01445ea5]
ERR #1: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x93c250) [0x7f2b01452250]
ERR #2: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x1633b9) [0x7f2b00c793b9]
ERR #3: /home/vcap/app/.java/lib/default/libj9prt29.so(+0x29978) [0x7f2b019f7978]
ERR #4: /lib/x86_64-linux-gnu/libc.so.6(+0x42520) [0x7f2b02133520]
ERR #5: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x4a43b9) [0x7f2b00fba3b9]
ERR #6: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x4a4447) [0x7f2b00fba447]
ERR #7: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x4a06bc) [0x7f2b00fb66bc]
ERR #8: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x3bc751) [0x7f2b00ed2751]
ERR #9: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x3bdf61) [0x7f2b00ed3f61]
ERR #10: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x32abfc) [0x7f2b00e40bfc]
ERR #11: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x3a7aaa) [0x7f2b00ebdaaa]
ERR #12: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x3affa7) [0x7f2b00ec5fa7]
ERR #13: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x697c1e) [0x7f2b011adc1e]
ERR #14: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x3a7002) [0x7f2b00ebd002]
ERR #15: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x7adfd5) [0x7f2b012c3fd5]
ERR #16: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x7afcf3) [0x7f2b012c5cf3]
ERR #17: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x59fc45) [0x7f2b010b5c45]
ERR #18: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x1795ff) [0x7f2b00c8f5ff]
ERR #19: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x17a631) [0x7f2b00c90631]
ERR #20: /home/vcap/app/.java/lib/default/libj9prt29.so(+0x2a3c9) [0x7f2b019f83c9]
ERR #21: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x178172) [0x7f2b00c8e172]
ERR #22: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x1784b8) [0x7f2b00c8e4b8]
ERR #23: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x1773db) [0x7f2b00c8d3db]
ERR #24: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x177742) [0x7f2b00c8d742]
ERR #25: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x1777f2) [0x7f2b00c8d7f2]
ERR #26: /home/vcap/app/.java/lib/default/libj9prt29.so(+0x2a3c9) [0x7f2b019f83c9]
ERR #27: /home/vcap/app/.java/lib/default/libj9jit29.so(+0x177bbb) [0x7f2b00c8dbbb]
ERR #28: /home/vcap/app/.java/lib/default/libj9thr29.so(+0xb3a3) [0x7f2b019c03a3]
ERR #29: /lib/x86_64-linux-gnu/libc.so.6(+0x94ac3) [0x7f2b02185ac3]
ERR Unhandled exception
ERR Type=Bus error vmState=0x000501ff
ERR J9Generic_Signal_Number=00000028 Signal_Number=00000007 Error_Value=00000000 Signal_Code=00000002
ERR Handler1=00007F2B01AA3620 Handler2=00007F2B019F7740 InaccessibleAddress=00007F2A95F15010
ERR RDI=00007F2AFC1A4C58 RSI=0000000000000000 RAX=00007F2A95F15000 RBX=00007F29569B5A74
ERR RCX=0000000000000000 RDX=0000000000000100 R8=0000000000000001 R9=0000000000000000
ERR R10=00007F29569B5A74 R11=00007F28C480F740 R12=0000000000013E8D R13=00000000000000B8
ERR R14=0000000000000000 R15=0000000000000001
ERR RIP=00007F2B00FBA3B9 GS=0000 FS=0000 RSP=00007F2AEE6C1250
ERR EFlags=0000000000010206 CS=0033 RBP=00007F2AEDEC7010 ERR=0000000000000006
ERR TRAPNO=000000000000000E OLDMASK=0000000000000000 CR2=00007F2A95F15010
ERR xmm0=0000000000000000 (f: 0.000000, d: 0.000000e+00)
ERR xmm1=0000000040e00000 (f: 1088421888.000000, d: 5.377519e-315)
ERR xmm2=00000000000000db (f: 219.000000, d: 1.082004e-321)
ERR xmm3=00000000000000d3 (f: 211.000000, d: 1.042479e-321)
ERR xmm4=0000000000000000 (f: 0.000000, d: 0.000000e+00)
ERR xmm5=00007f28c4810970 (f: 3296790784.000000, d: 6.907684e-310)
ERR xmm6=0000000002edf718 (f: 49149720.000000, d: 2.428319e-316)
ERR xmm7=00007f28c47f6af0 (f: 3296684800.000000, d: 6.907684e-310)
ERR xmm8=ffffff00ffffff00 (f: 4294967040.000000, d: -nan)
ERR xmm9=7575757575757575 (f: 1970632064.000000, d: 6.444099e+257)
ERR xmm10=757575767575757d (f: 1970632064.000000, d: 6.444104e+257)
ERR xmm11=0000015200000151 (f: 337.000000, d: 7.172346e-312)
ERR xmm12=0000013d00000140 (f: 320.000000, d: 6.726727e-312)
ERR xmm13=000001380000013f (f: 319.000000, d: 6.620627e-312)
ERR xmm14=0000000008001800 (f: 134223872.000000, d: 6.631540e-316)
ERR xmm15=000001420000013b (f: 315.000000, d: 6.832826e-312)
ERR Module=/home/vcap/app/.java/lib/default/libj9jit29.so
ERR Module_base_address=00007F2B00B16000
ERR Method_being_compiled=com/mongodb/internal/async/function/RetryState.doAdvanceOrThrow(Ljava/lang/Throwable;Ljava/util/function/BiFunction;Ljava/util/function/BiPredicate;Z)V
ERR Target=2_90_20240716_818 (Linux 5.15.0-113-generic)
ERR CPU=amd64 (16 logical CPUs) (0xbc29df000 RAM)
ERR ----------- Stack Backtrace -----------
ERR _ZN12TR_IProfiler17findOrCreateEntryEimb+0x179 (0x00007F2B00FBA3B9 [libj9jit29.so+0x4a43b9])
ERR _ZN12TR_IProfiler15profilingSampleEmmbbj.localalias+0x47 (0x00007F2B00FBA447 [libj9jit29.so+0x4a4447])
ERR _ZN12TR_IProfiler12setCallCountEP20TR_OpaqueMethodBlockiiPN2TR11CompilationE.localalias+0x1c (0x00007F2B00FB66BC [libj9jit29.so+0x4a06bc])
ERR _ZN21TR_J9EstimateCodeSize20realEstimateCodeSizeEP13TR_CallTargetP12TR_CallStackbRN2TR6RegionE+0x1861 (0x00007F2B00ED2751 [libj9jit29.so+0x3bc751])
ERR _ZN21TR_J9EstimateCodeSize16estimateCodeSizeEP13TR_CallTargetP12TR_CallStackb.localalias+0x41 (0x00007F2B00ED3F61 [libj9jit29.so+0x3bdf61])
ERR _ZN19TR_EstimateCodeSize17calculateCodeSizeEP13TR_CallTargetP12TR_CallStackb+0xac (0x00007F2B00E40BFC [libj9jit29.so+0x32abfc])
ERR _ZN28TR_MultipleCallTargetInliner13weighCallSiteEP12TR_CallStackP11TR_CallSitebb+0x1fa (0x00007F2B00EBDAAA [libj9jit29.so+0x3a7aaa])
ERR _ZN28TR_MultipleCallTargetInliner17inlineCallTargetsEPN2TR20ResolvedMethodSymbolEP12TR_CallStackP24TR_InnerPreexistenceInfo+0x1757 (0x00007F2B00EC5FA7 [libj9jit29.so+0x3affa7])
ERR _ZN14TR_InlinerBase15performInliningEPN2TR20ResolvedMethodSymbolE+0xae (0x00007F2B011ADC1E [libj9jit29.so+0x697c1e])
ERR _ZN10TR_Inliner7performEv+0x142 (0x00007F2B00EBD002 [libj9jit29.so+0x3a7002])
ERR _ZN3OMR9Optimizer19performOptimizationEPK20OptimizationStrategyiii.localalias+0x855 (0x00007F2B012C3FD5 [libj9jit29.so+0x7adfd5])
ERR _ZN3OMR9Optimizer8optimizeEv+0x1b3 (0x00007F2B012C5CF3 [libj9jit29.so+0x7afcf3])
ERR _ZN3OMR11Compilation7compileEv+0xa25 (0x00007F2B010B5C45 [libj9jit29.so+0x59fc45])
ERR _ZN2TR28CompilationInfoPerThreadBase7compileEP10J9VMThreadPNS_11CompilationEP17TR_ResolvedMethodR11TR_J9VMBaseP19TR_OptimizationPlanRKNS_16SegmentAllocatorE+0x4bf (0x00007F2B00C8F5FF [libj9jit29.so+0x1795ff])
ERR _ZN2TR28CompilationInfoPerThreadBase14wrappedCompileEP13J9PortLibraryPv+0x381 (0x00007F2B00C90631 [libj9jit29.so+0x17a631])
ERR omrsig_protect+0x239 (0x00007F2B019F83C9 [libj9prt29.so+0x2a3c9])
ERR _ZN2TR28CompilationInfoPerThreadBase7compileEP10J9VMThreadP21TR_MethodToBeCompiledRN2J917J9SegmentProviderE+0x372 (0x00007F2B00C8E172 [libj9jit29.so+0x178172])
ERR _ZN2TR24CompilationInfoPerThread12processEntryER21TR_MethodToBeCompiledRN2J917J9SegmentProviderE+0x128 (0x00007F2B00C8E4B8 [libj9jit29.so+0x1784b8])
ERR _ZN2TR24CompilationInfoPerThread14processEntriesEv+0x35b (0x00007F2B00C8D3DB [libj9jit29.so+0x1773db])
ERR _ZN2TR24CompilationInfoPerThread3runEv+0x42 (0x00007F2B00C8D742 [libj9jit29.so+0x177742])
ERR _Z30protectedCompilationThreadProcP13J9PortLibraryPN2TR24CompilationInfoPerThreadE+0x82 (0x00007F2B00C8D7F2 [libj9jit29.so+0x1777f2])
ERR omrsig_protect+0x239 (0x00007F2B019F83C9 [libj9prt29.so+0x2a3c9])
ERR _Z21compilationThreadProcPv+0x17b (0x00007F2B00C8DBBB [libj9jit29.so+0x177bbb])
ERR thread_wrapper+0x163 (0x00007F2B019C03A3 [libj9thr29.so+0xb3a3])
ERR (0x00007F2B02185AC3 [libc.so.6+0x94ac3])
ERR clone+0x44 (0x00007F2B02216A04 [libc.so.6+0x125a04])
ERR ---------------------------------------

```
